packages/edit-widgets/src/components/layout/index.js

```diff
@@ -38,6 +38,7 @@ function Layout( { blockEditorSettings } ) {
 							content={
 								<>
 									<Notices />
+									<Popover.Slot name="block-toolbar" />
```

Comment:
I'm wondering if we want one slot per widget area here or a single slot for the whole page. It's not clear how we should reason about this particular slot, where to put it if you're building your own editor. (maybe needs some docs)

Also, do we want to expose it as a `BlockToolbarSlot` in @wordpress/block-editor package?

Reply:
That's the thing... I was hoping that we could build it into WritingFlow or BlockList (together with WritingFlow or something) to avoid exposing it. But maybe there's no other way. I also had an earlier idea to unify it with the top toolbar, in which case exposing it maybe makes more sense.

Reply:
Ok, got it. I'm fine if we wait more and see if we can consolidate.

Reply:
What about the widgets screen question?

Reply:
I don't know so much about the widgets. Did I currently add one slot for all areas? If that works, that seems fine? What do you think?

Reply:
I think the tabbing order might not be great with this solution but at least the toolbar is showing.

Reply:
Oh, I guess it might be better to have one per area then. Let's iterate on it. Worth noting that previously, it wasn't possible to tab to the toolbar at all because it was mounted inside WritingFlow.